database/migrations/2020_08_29_182717_create_import_items_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateImportItemsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('import_items', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->string('name');
```

Comment:
We could add the serial number here as an optional column. I kind of remember that this should be added to laptops.

Reply:
Maybe it would be enough to get the serial number in a different input field, and then adding it to the name (eg. in parantheses) when saving.

Reply:
I think it would be clearer to save it separately then as well.